tests/e2e/vsphere.go

```diff
@@ -936,3 +936,103 @@ func waitForHostConnectionState(ctx context.Context, addr string, state string)
 	framework.Logf("The host's %s last seen state before returning is : %s", addr, output)
 	return waitErr
 }
+
+// deleteVolumeSnapshotInCNS Call deleteSnapshots API
+func (vs *vSphere) deleteVolumeSnapshotInCNS(fcdID string, snapshotId string) error {
+	ctx, cancel := context.WithCancel(context.Background())
+	defer cancel()
+
+	// Connect to VC
+	connect(ctx, vs)
```

Comment:
why we need it here as well? it will be done in beforeach bootstrap call right

Reply:
this is needed to counter the not authenticated issue when the the function is called individually.